client-tez/src/main/java/org/apache/tez/runtime/library/common/shuffle/orderedgrouped/RssShuffleScheduler.java

```diff
@@ -493,6 +494,7 @@ public void start() throws Exception {
   }
 
   @Override
+  @SuppressWarnings("all")
```

Comment:
```suggestion
  @SuppressWarnings("NN_NAKED_NOTIFY")
```

I think NN_NAKED_NOTIFY is enough?

Reply:
> I think NN_NAKED_NOTIFY is enough?

ok, I corrected it and pushed it.